pkg/tnf/test.go

```diff
@@ -33,6 +33,13 @@ const (
 	FAILURE
 )
 
+// EXITCODEMAP maps a test result value to a more appropriate Unix return code.
+var EXITCODEMAP = map[int]int{
```

Comment:
Please use the convention of MixedCaps as described here: https://golang.org/doc/effective_go#mixed-caps.

I.e.,

`ExitCodeMap`